relevanceai/api/endpoints/deployables/dashboard.py

```diff
@@ -0,0 +1,138 @@
+from abc import ABC, abstractmethod
+from typing import Collection
+
+from relevanceai.api.endpoints.datasets.datasets import DatasetsClient
+from relevanceai.base import _Base
+from .deployables import Deployable
+
+
+class Dashboard(ABC, _Base):
```

Comment:
I like this metaclass. Properly can also use the following: 
- delete
- create
-

Reply:
Actually, the delete feature was something I was going to bring up for discussion after I got this thing to work. Without getting into too many details here, I think it might be better for an individual dashboard to not have a delete method and have any delete calls call from the deployables endpoint.